iulianIachimov/src/main/java/datacollectiontask/Models/Order.java

```diff
@@ -0,0 +1,53 @@
+package datacollectiontask.Models;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    public long id;
+    public LocalDateTime date;
+    public User user;
+    public List<Item> items;
+
+    private static long idCounter;
```

Comment:
please create a method to generate an ID of type UUID

Reply:
It was implemented.